cmd/gen-go-sample/testdata/sample.want

```diff
@@ -1,3 +1,26 @@
+// Copyright 1265003508 Google LLC
```

Comment:
Lol where is the year 1265003508 coming from?

Could you update its source to a 4-digit year? :)

Reply:
I hash coded a string :)